.circleci/config.yml

```diff
@@ -121,6 +139,9 @@ workflows:
       - go1.11-alpine:
           requires:
             - get_source
+      - go1.11-macos:
```

Comment:
Are we able to add a MacOS job into a workflow that is usually running on Linux?